src-tauri/src/textures.rs

```diff
@@ -21,19 +24,19 @@ pub async fn extract_textures(app_handle: tauri::AppHandle, textures_array: Vec<
     .unwrap()
     .join("data/texture_replacements");
 
-  let target_dir = PathBuf::from(text_dir.clone()); // Doesn't need to exist
-
-  // for path in textures_array {
-  //     println!("Extracting texture pack: {:?}", path.clone());
-
-  //     let archive: Vec<u8> = fs::read(&path.clone()).unwrap();
-  //     // The third parameter allows you to strip away toplevel directories.
-  //     // If `archive` contained a single directory, its contents would be extracted instead.
-  //     match zip_extract::extract(Cursor::new(archive), &target_dir, true) {
-  //         Ok(_) => continue,
-  //         Err(err) => println!("{:?}", err),
-  //     }
-  // }
+  for path in textures_array {
+    println!(
+      "Not extracting (not yet implemented) texture pack to {}: {path:?}",
+      text_dir.display(),
+    );
```

Comment:
This was just a trick to use the variables and not have to comment it all out to avoid warnings.